bot.py

```diff
@@ -0,0 +1,65 @@
+from twitchio.ext import commands
+import asyncpg
+import openai
+import os
+import dotenv
+
+dotenv.load_dotenv()
+
+async def verse_generation(text):
+    openai.api_key = os.getenv("OPENAPI_KEY")
+
+    verse = openai.ChatCompletion.create(
+        model="gpt-3.5-turbo", 
+        messages=[{
+            "role": "user", 
+            "content": f"Create a verse that rhymes up to 3 lines using the following message: {text}"
+        }]
+    )
+
+    return verse.choices[0].message.content
+
+# Database connection
+async def db_connection():
+    connection = await asyncpg.connect(
+        user=os.getenv("POSTGRES_USERNAME"), 
+        password=os.getenv("POSTGRES_PASSWORD"),
+        database=os.getenv("POSTGRES_DB_NAME"), 
+        host='localhost'
+    )
+    
+    return connection
+
+# This function persists the messages in the database when called
+async def store_message(msg):
+    connection = await db_connection()
```

Comment:
do asyncpg deals with connection pools and such? or are we always creating a new connection to the database for every message we process?